src/components/ActionBar/ActionBar.spec.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { ActionBar } from './ActionBar';
+import { defaultAggs } from '../../reducers/aggs/aggs';
+import { defaultQuery } from '../../reducers/query/query';
+import { merge } from '../../testUtils/functionHelpers';
+import {
+  testRender as render,
+  fireEvent,
+  screen,
+} from '../../testUtils/test-utils';
+import * as dataExportActions from '../../actions/dataExport';
+import * as utils from '../../utils';
+
+describe('ActionBar', () => {
+  const renderComponent = (newAggsState, newQueryState) => {
+    merge(newAggsState, defaultAggs);
+    merge(newQueryState, defaultQuery);
+
+    const data = {
+      aggs: newAggsState,
+      query: newQueryState,
+    };
+
+    render(<ActionBar />, {
+      preloadedState: data,
+    });
+  };
+
+  let gaSpy;
+  const { location } = window;
+  beforeEach(() => {
+    gaSpy = jest.spyOn(utils, 'sendAnalyticsEvent');
+    delete window.location;
+    // provide an empty implementation for window.assign
+    window.location = {
+      assign: jest.fn(),
+      href: 'http://ccdb-website.gov',
+    };
+  });
+
+  afterEach(() => {
+    window.location = location;
+  });
+
+  test('rendering', () => {
+    const aggs = {
+      doc_count: 100,
+      total: 10,
+    };
+
+    const query = {
+      tab: 'Map',
+    };
+
+    const dataExportSpy = jest
+      .spyOn(dataExportActions, 'showExportDialog')
+      .mockImplementation(() => jest.fn());
+    renderComponent(aggs, query);
+
+    expect(
+      screen.getByText('Showing 10 matches out of 100 total complaints')
+    ).toBeInTheDocument();
+    const buttonExport = screen.getByRole('button', { name: 'Export data' });
+    expect(buttonExport).toBeInTheDocument();
+    fireEvent.click(buttonExport);
+    expect(dataExportSpy).toHaveBeenCalledTimes(1);
+
+    const buttonPrint = screen.getByRole('button', { name: 'Print' });
+    expect(buttonPrint).toBeInTheDocument();
+    fireEvent.click(buttonPrint);
+
+    expect(window.location.assign).toHaveBeenCalledWith(
+      'http://ccdb-website.gov&isPrintMode=true&' + 'isFromExternal=true'
```

Comment:
Why is this broken up into two different strings?

Reply:
If the order of the the url isn't guaranteed, you could use
```
expect(window.location.assign).toHaveBeenCalledWith(
      expect.stringContaining('http://ccdb-website.gov&isPrintMode=true&') &&
        expect.stringContaining('isFromExternal=true')
    );
```

Reply:
i missed some things when i copy pasted.  i think the linter probably changed and didn't fix my formatting.

Reply:
updated.  The order shouldn't change.  The original code also had this linting warning.